mv_avs_dir.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
s/bash/sh/
as `declare` bash built-in function is not needed (I'll describe the reason at the below comment).